_posts/2018-11-20-mongodb-europe-018.md

```diff
@@ -0,0 +1,411 @@
+---
+layout: post
+authors: [chris_de_bruyne, nick_van_hoof, jan_van_der_veken]
+title: "MongoDB Europe 2018"
+image: /img/2018-11-08-mongodb-europe-2018/main-image-mdbe.png
+tags: [Development,MongoDB,DBA,Data,Kubernetes,Conference]
+category: Development
+comments: true
+---
+
+> MongoDB Europe is a yearly conference where MongoDB shows off their latest features and new products.
+> This year the venue took place in Old Billingsgate Walk, London.
+> Jan, Nick and Chris wrote this blog post to summarize several of the given sessions.
+
+
+# Table of contents
+1. [Atlas](#atlas)
+2. [MongoDB University](#mongodb-university)
+3. [Compass Aggregation Pipeline Builder](#compass-aggregation-pipeline-builder)
+4. [Common query mistakes](#common-query-mistakes)
+5. [Stitch](#stitch)
+6. [Meet the experts](#meet-the-experts)
+7. [Streaming data pipelines with MongoDB and Kafka at AO](#streaming-data-pipelines-with-mongodb-and-kafka-at-ao)
+8. [MongoDB Enterprise Operator for Kubernetes at Amadeus](#mongodb-enterprise-operator-for-kubernetes-at-amadeus)
+9. [MongoDB Charts](#mongodb-charts)
+10. [Closing](#closing)
+
+# Atlas
+([MongoDB Atlas for your Enterprise](https://sched.co/FmAF){:target="_blank" rel="noopener noreferrer"}, Vladislava Stevanovic & Veronica Tudor)
+
+With Atlas, MongoDB brings us DBaaS (Database As A Service).
```

Comment:
This section reads a bit odd as the sentences are all structured identically: someone verbed something.
Try to shuffle it around a bit for a better reading experience.